main

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+require './app'
+
```

Comment:
Please add the `.rb` extension as this is also a Ruby file. The project does not run without adding an extension to this file.